src/sidebar/components/search-status-bar.js

```diff
@@ -22,46 +28,57 @@ const countVisibleAnns = annThread => {
  * any search results were found.
  * */
 function SearchStatusBar({ rootThread }) {
-  const {
-    directLinkedGroupFetchFailed,
-    filterQuery,
-    selectedAnnotationMap,
-    selectedTab,
-  } = useStore(store => ({
+  const actions = useStore(store => ({
+    clearSelection: store.clearSelection,
+  }));
+  const rootState = useStore(store => ({
     directLinkedGroupFetchFailed: store.getRootState().directLinked
       .directLinkedGroupFetchFailed,
     filterQuery: store.getRootState().selection.filterQuery,
     selectedAnnotationMap: store.getRootState().selection.selectedAnnotationMap,
     selectedTab: store.getRootState().selection.selectedTab,
   }));
-  const clearSelection = useStore(store => store.clearSelection);
-  const filterActive = !!filterQuery;
-  const annotationCount = useStore(store => store.annotationCount());
-  const noteCount = useStore(store => store.noteCount());
+  const selectors = useStore(store => ({
```

Comment:
The `selectors` naming is misleading here IMO because "selectors" are functions whereas this object contains the current _values_ resulting from calling those functions. Also, there isn't a good reason to differentiate these values from the ones obtained by direct accesses into the store state.

My suggestion here would be just to merge this `useStore` call with the one above and either:

- Name the resulting variable `state` or `storeState` OR
- Use destructuring to extract the resulting values into variables:
   ```js
   const { someValue } = useStore(store => ({ someValue: store.getSomeValue() }))
   ```

The advantage of the latter pattern is that you'll get feedback immediately from the linter if a value becomes unused because the variable will be unused.